src/storage/Cargo.toml

```diff
@@ -61,14 +69,27 @@ postgres-protocol = { version = "0.6.5" }
 prometheus = { version = "0.13.3", default-features = false }
 prost = { version = "0.11.3", features = ["no-recursion-limit"] }
 rand = "0.8.5"
-rdkafka = { version = "0.29.0", features = ["cmake-build", "ssl-vendored", "libz-static", "zstd"] }
+rdkafka = { version = "0.29.0", features = [
+  "cmake-build",
+  "ssl-vendored",
+  "libz-static",
+  "zstd",
+] }
 regex = { version = "1.7.0" }
-rocksdb = { git = "https://github.com/MaterializeInc/rust-rocksdb", branch = "master", default-features = false, features = ["snappy", "zstd", "lz4"] }
+reqwest = { version = "0.11", features = ["blocking"] }
+rocksdb = { git = "https://github.com/MaterializeInc/rust-rocksdb", branch = "master", default-features = false, features = [
+  "snappy",
+  "zstd",
+  "lz4",
+] }
 seahash = "4"
 serde = { version = "1.0.152", features = ["derive"] }
 serde_json = { version = "1.0.89" }
 sha2 = "0.10.6"
-timely = { version = "0.12.0", default-features = false, features = ["bincode"] }
+tar = "0.4.38"
+timely = { version = "0.12.0", default-features = false, features = [
+  "bincode",
+] }
```

Comment:
Do you have some TOML formatter installed? This is an unusual formatting, and not our standard style.

Reply:
Yeah, will revert this.